shell/platform/windows/flutter_window_win32_unittests.cc

```diff
@@ -118,8 +118,7 @@ class MockFlutterWindowWin32 : public FlutterWindowWin32 {
                void(double, double, FlutterPointerDeviceKind, int32_t, UINT));
   MOCK_METHOD2(OnPointerLeave, void(FlutterPointerDeviceKind, int32_t));
   MOCK_METHOD0(OnSetCursor, void());
-  MOCK_METHOD4(OnScroll,
-               void(double, double, FlutterPointerDeviceKind, int32_t));
```

Comment:
Why was `OnScroll` removed?

Reply:
This is because that function can't be mocked as the non-mocked version needs to be called as part of the test.